src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_command_type.py

```diff
@@ -3,35 +3,63 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 #---------------------------------------------------------------------------------------------
 
+"""
+Utiilty methods for register storage data plane commands
+"""
+
+from collections import namedtuple
 from azure.cli.core.commands import create_command, command_table
 from azure.cli.command_modules.storage._validators import validate_client_parameters
 
+
+def _register_data_plane_arguments(command, group_name):
+    """
+    Register the four parameters which can be used to represent the storage account credential.
+    A optinal prefix can be given to the four parameters. It useful in the scenarios where
+    multilpe storage accounts are involved.
+    """
+
+    get_option_name = lambda name: ('--%s' % name).replace('_', '-')
+
+    ArgDef = namedtuple("ArgDef", ['name', 'validator', 'help'])
+
+    basic_definitions = [
+        ArgDef('account_name', None,
+               'Storage account name. Must be used in conjunction with either '
+               'storage account key or a SAS token. Environment variable: '
+               'AZURE_STORAGE_ACCOUNT'),
```

Comment:
If this is essentially a factory for storage connection parameters, you would also need to update the environment variable names.